cloud/scope/cluster.go

```diff
@@ -114,6 +114,7 @@ func (s *ClusterScope) PublicIPSpecs() []azure.PublicIPSpec {
 		{
 			Name:    s.Network().APIServerIP.Name,
 			DNSName: s.Network().APIServerIP.DNSName,
+			IsIPV6:  false, // currently azure requires a ipv4 lb rule to enable ipv6
```

Comment:
can we link to an issue or docs here? "currently" might not be relevant anymore when someone reads this next year